pkg/fluentd/fluentd_test.go

```diff
@@ -90,6 +91,45 @@ func TestCreateFluentdContainer(t *testing.T) {
 	assertion.Equal("/u01/data/docker/containers", fluentd.VolumeMounts[2].MountPath, "Fluentd container volume mounts mount path not equal to expected value")
 	assertion.Equal("datadockercontainers", fluentd.VolumeMounts[2].Name, "Fluentd container volume mounts name not equal to expected value")
 	assertion.Equal(true, fluentd.VolumeMounts[2].ReadOnly, "Fluentd container volume mounts read only flag not equal to expected value")
+
+	// In addition test that when Dev mode is set we get the expected elastic search host
+	os.Setenv("SINGLE_SYSTEM_VMI", "true")
+	fluentd = CreateFluentdContainer("test-binding", "test-component")
+	os.Unsetenv("SINGLE_SYSTEM_VMI")
+
+	assertion.Equal("fluentd", fluentd.Name, "Fluentd container name not equal to expected value")
+	assertion.Equal(2, len(fluentd.Args), "Fluentd container args count not equal to expected value")
```

Comment:
Maybe should fail if len is wrong to avoid potential out of bounds condition in subsequent checks.

Reply:
sure, I'm assuming that applies to the existing code that I copied those checks from above (and any similar existing test code).

I can change these for this, but we maybe should open a task to review the existing tests for similar

Reply:
I opened a separate task to review other existing tests similarly (as a separate task)